packages/react-core/src/components/Modal/ModalContent.tsx

```diff
@@ -37,8 +37,6 @@ export interface ModalContentProps {
   footer?: React.ReactNode;
   /** Action buttons to add to the standard Modal Footer, ignored if `footer` is given */
   actions?: any;
-  /** Flag to indicate that the Footer content is left aligned */
-  isFooterLeftAligned?: boolean;
```

Comment:
These changes are addressed as part of https://github.com/patternfly/patternfly-react/pull/4017

Reply:
Good catch, I merged #4017 and am fixing the conflicts on this PR now.

The 79 commits is correct.